model/autosklearn/examples/autoclassifier/accuracy.sh

```diff
@@ -9,4 +9,5 @@ dffml accuracy \
     SepalWidth:float:1 \
     PetalLength:float:1 \
     PetalWidth:float:1 \
+  -accuracy-scorer=mse \
```

Comment:
We usually don't use `=`, so let's not use it for this.

```suggestion
  -accuracy-scorer mse \
```

Side note, you can look for commands that will need to be changed to add the `-accuracy-scorer` using a command like:

```console
$ git grep -A 25 -E 'dffml train|dffml accuracy|dffml predict' | python -c 'import sys; print("--".join([i for i in sys.stdin.read().split("--") if not "accuracy-scorer" in i]).strip())'
```

However, that will only show you `.sh` commands, you'll want maybe something like

```console
$ git grep -A 25 -E 'train|accuracy|predict'
```

To look at it all, but that's going to give you a ton of results to filter though